app/src/main/java/org/fossasia/openevent/app/core/event/list/EventsListAdapter.java

```diff
@@ -77,8 +80,9 @@ public EventRecyclerViewHolder onCreateViewHolder(ViewGroup parent, int viewType
         EventLayoutBinding binding = EventLayoutBinding.inflate(layoutInflater, parent, false);
         EventRecyclerViewHolder eventRecyclerViewHolder = new EventRecyclerViewHolder(binding);
 
-        eventRecyclerViewHolder.onItemLongClick(eventsPresenter::toolbarEditMode);
```

Comment:
As you are removing the option to edit event on long press, make sure that the code related to it such as the toolbarEditMode( ) in the EventsPresenter also gets removed.